app/views/providers/_user_contact.html.erb

```diff
@@ -8,7 +8,7 @@
     <%= image_tag user.avatar %>
   </div>
   <p class="name"> <%= user.nickname %> </p>
-  <% if current_user.conversation?(user, @provider)%>
+  <% if !session[:usrre_id].nil? && current_user.conversation?(user, @provider)  %>
```

Comment:
i assume that the first condition should exclude the case where there is no user logged in, but in my console this variable is nil even when i m logged in. so i would kind of think we never reach whats in the if? i guess its still correct, but i would like to understand how it works! :)